src/branch-utils.js

```diff
@@ -1,13 +1,7 @@
-import { config as configStore } from './stores';
-
-let branchPattern;
-configStore.subscribe(value => ({ branchPattern } = value));
-
 const branchToken = '{branch}';
 const styleToken = '{style}';
 
-const createBranchUrl = (branchName, selectedStyle) => {
-  const { pattern } = branchPattern;
+const createBranchUrl = (pattern, branchName, selectedStyle) => {
```

Comment:
This function just constructs the URL now so this file feels less essential to break out like this, but I left it in its own file for now anyway. If we decide to take a different approach to this PR to not pass around `pattern`, I imagine we'd have to also include some kind of hash parsing for branch urls as a function in this file too.